README.md

```diff
@@ -1 +1,117 @@
-# Math_Magicians
\ No newline at end of file
+<div align="center">
+  <h3><b>Math Magicians</b></h3>
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+[📗 Table of Contents](#-table-of-contents)
+
+- [📖 Math Magicians ](#-math-magicians-)
+  - [🛠 Built With ](#-built-with-)
+    - [Tech Stack ](#tech-stack-)
+    - [Key Features ](#key-features-)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Usage](#usage)
+    - [Run tests](#run-tests)
+  - [👥 Authors ](#-authors-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [📝 License ](#-license-)
+
+# 📖 Math Magicians <a name="about-project"></a>
+
+"Math magicians" is a website for all fans of mathematics. It is a Single Page App (SPA) that allows users to:
+
+Make simple calculations.
+Read a random math-related quote.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+- HTML
+- CSS
+- React
+
+### Key Features <a name="key-features"></a>
+
+- [ ] **Make simple calculations.**
+- [ ] **Read a random math-related quote.**
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+- a browser
+- git
+- nodejs
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```sh
+  cd my-folder
+  git clone https://github.com/ybadr99/math_magicians.git
+  npm install
+```
+
+### Usage
+
+To run the project, execute the following command:
+
+```sh
+  npm run dev
+```
+
+### Run tests
+
+To run tests, run the following commands:
+
+```sh
+  npx stylelint "**/*.{css,scss}"
+  npx hint .
+  npx eslint .
+```
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+> Yousef Badr
+> 👤 **Author1**
+
+- GitHub: [@ybadr99](https://github.com/ybadr99)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/yousef-mohamed-badr/)
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Feel free to check the [issues page](https://github.com/ybadr99/leaderboard/issues).
+Contributions, issues, and feature requests are welcome!
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project:thumbsup:
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+> I would like to thank [Leaderboard API](https://microverse.notion.site/Leaderboard-API-service-24c0c3c116974ac49488d4eb0267ade3)
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE.md) licensed.
```

Comment:
- [ ] Great job working on this milestone. Your license link is broken and given Error 404 when clicked. It is not professional to have broken links in the README. Kindly fix this.
![image](https://github.com/ybadr99/Math_Magicians/assets/62849872/274cd3bf-6b8d-46a0-9478-027110a5b939)